doc/config.toml

```diff
@@ -226,7 +226,7 @@ url = "http://www.apache.org/security/"
 # End user relevant links. These will show up on left side of footer and in the community page if you have one.
 [[params.links.user]]
   name = "User mailing list"
-  url = "mailto:user@avro.apache.org"
+  url = "https://lists.apache.org/list.html?user@avro.apache.org"
```

Comment:
Do we want to keep the user stay on the current page and open the lists.a.o in a new browser tab ?
If YES then we will have to add `target="_blank"` to the Markdown page where this `url` is rendered.

Reply:
I don't think opening is the same page or tab is unexpected!  If anyone feels otherwise, we can probably revisit this decision!